### ISSUE TITLE
Make GitHub packages world readable

### ISSUE THREAD

**Issue by physikerwelt** [WDscholia/scholia#2204]
**Is your feature request related to a problem? Please describe.**
Currently, GitHub packages are created but are not public.

**Describe the solution you'd like**
Packages are available for download.

**Describe alternatives you've considered**
---

**Additional context**
You can make the packages public as described here https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility

**Comment by fnielsen** [comment]
What would GitHub packages be for us? I am trying to understand whether it is related to Docker or Pip.

**Comment by physikerwelt** [comment]
It is related to docker. Currently one can do for our fork
```
docker pull ghcr.io/mardi4nfdi/scholia:nightly
```
but not for the offical repo.

**Comment by egonw** [comment]
@physikerwelt, if I understand it correctly, it allows us to publish Scholia as docker easily, correct? Is `ghcr.io` like DockerHub?

**Comment by physikerwelt** [comment]
Yes, ghcr.io is like GitHub. Moreover, the images are already pushed via the GitHub CI to ghcr; it is just not publicly accessible.

**Comment by egonw** [comment]
@physikerwelt, yeah, I can see it is "private":

![image](https://github.com/WDscholia/scholia/assets/26721/c334b295-29c6-492a-8f10-0891059fd12b)

But I did not find so far where to make it public :(

**Comment by egonw** [comment]
Oh, I found it now :)

![image](https://github.com/WDscholia/scholia/assets/26721/51e147bf-bc40-4fa2-92ee-6d893f0ecbfd)

@fnielsen, are you okay to make it public?
